subsystem-drive/dto/drive-dto.hpp

```diff
@@ -1,33 +1,46 @@
 #pragma once
+
 namespace sjsu::drive
 {
     // note: postfix commands and arguments can be seperated into different files
     struct drive_commands
     {
-        int angle = 0; // -180 <--> 180
+        char mode = 'D';
         int speed = 0; // -100 <--> 100
+        int angle = 0; // -180 <--> 180
         int wheel_orientation = 0;
-        char mode = 'D';
+
+        void Print()
+        {
+            printf("Cmds\tMode\tSpeed\tAngle\tOri\n");
+            printf("\t%c\t%d\t%d\t%d\n", mode, speed, angle, wheel_orientation);
```

Comment:
I like the idea of a print function in each class, keep it easy to print what information we really want rather then a big blob of information all at once.